idd/Energy+.idd.in

```diff
@@ -76130,6 +76130,39 @@ FaultModel:HumidistatOffset,
        \note Required field for Type ThermostatOffsetDependent
        \type object-list
        \object-list ThermostatOffsetFaults   
+	   
+FaultModel:Fouling:AirFilter,
+   \memo This object describes fault of dirty air filters
+   A1, \field Name
+       \required-field
+       \type alpha
```

Comment:
And I know it's redundant, but you might add a \note to the Name field.  Just so the window isn't blank when a user sees this object in IDF-Editor, for example.